db/models/assessment_record/assessment_records.py

```diff
@@ -0,0 +1,82 @@
+from sqlalchemy import Index
+from db import db
+from db.models.assessment_record.enums import Status
+from sqlalchemy import cast
+from sqlalchemy import Computed
+from sqlalchemy import func
+from sqlalchemy.dialects.postgresql import ENUM
+from sqlalchemy.dialects.postgresql import JSONB
+from sqlalchemy.dialects.postgresql import TEXT
+from sqlalchemy.dialects.postgresql import UUID
+
+
+class AssessmentRecords(db.Model):
+    __tablename__ = "assessment_records"
+
+    application_id = db.Column(
+        "application_id", UUID(), primary_key=True
+    )
+
+    short_id = db.Column("short_id", db.Text(), nullable=False)
+
+    type_of_application = db.Column(
+        "type_of_application", db.Text(), index=True, nullable=False
```

Comment:
Can we make this a string type rather than text?

Reply:
How about the postgres specific TEXT since it's a infinite varchar by default like String?

Reply:
I thought String mapped to varchar ref: https://docs.sqlalchemy.org/en/14/core/type_basics.html#sqlalchemy.types.String so not infinite, and better for this kind of case (short strings)?

I think Text would be fine for things like comment body but String for fields like this.

Reply:
Ok, I will swap for a String(256)!

Reply:
(255 I mean)